angelsbioprocessing/locale/en/bio-processing.cfg

```
@@ -1,538 +1,538 @@
-bio-temperate-farming=Temperate Farming
-bio-temperate-farm=Temperate Farming Upgrade
-bio-swamp-farming=Swamp Farming
-bio-swamp-farm=Swamp Farming Upgrade
-bio-nutrient-paste=Nutrient-Paste
-bio-pressing=Plant Pressing
-bio-pressing-fish=Fish Pressing
-bio-fermentation=Fermentation Process
-bio-plastic=Bio Plastics
-
-bio-paper=Paper Making
-bio-wood-processing=Wood Processing
-bio-arboretum=Tree Arboretum
-bio-arboretum-temperate=Temperate Tree Arboretum
-bio-arboretum-swamp=Swamp Tree Arboretum
-bio-arboretum-desert=Desert Tree Arboretum
-
-bio-refugium-fish=Fish Refugium
-bio-refugium-hatchery=Hatchery
-bio-refugium-butchery=Butchery
-bio-refugium-puffer=Puffer Refugium
-bio-refugium-biter=Biter Refugium
-bio-refugium-hogger=Hogger Refugium
-
-[item-description]
-composter="Used to turn any Organic Material into compost"
-solid-saw="He saw, he cut and he shattered"
-
-temperate-1="Wil. Plant from a place where no man has gone before"
-temperate-2="Fluffy. Makes you want to go to bed"
-temperate-3="Sour. Best served in a hot sauce"
-temperate-4="Adaptive. Can be used for a variety of things"
-temperate-5="Tasty. Very generous range of flavours"
-
-desert-1="Sweet. Gets you all fuzzy and warm around your heart"
-desert-2="Organized. Grows in ordered lines"
-desert-3="Productive. Will always arrange itself in orderly lines"
-desert-4="Zealous. Always dynamic"
-desert-5="Salty. Has to be handled with care"
-
-swamp-1="Nutty. Makes you slightly mad"
-swamp-2="Lively. Seem to be growing disregarding the conditions"
-swamp-3="Commanding. Don't use too much"
-swamp-4="Binary. It grows or it does not, nothing in between"
-swamp-5="Spicy. Seems to yell at you"
-
-solid-nuclear-fertilizer="This doesn't sound healty... We might start to glow at night!"
-
-raw-fish="A friendly reminder to who knew Factorio before an Angel stopped by"
-alien-fish-1-raw="The least deadly fish on this planet"
-alien-fish-2-raw="Oblivious of their surroundings, very frugal beings"
-alien-fish-3-raw="Lovely Fish at first sight, but don't anger them"
-
-bio-raw-meat="It tastes like chicken..."
-
-[entity-description]
-bio-refugium-biter="Fenced in safely from the outside world, biters can be grown here"
-bio-refugium-fish="More than just decoration, can be made home to a variety of fish"
-bio-refugium-hogger="Dirt digging Hogger feel right at home here"
-bio-refugium-puffer="It stinks, but Puffers feel right at home in this gas filled tower"
-
-[recipe-description]
-temperate-garden-a=Temperate Seed Extraction
-temperate-garden-b=Advanced Temperate Seed Extraction
-
-desert-garden-a=Desert Seed Extraction
-desert-garden-b=Advanced Desert Seed Extraction
-
-swamp-garden-a=Swamp Seed Extraction
-swamp-garden-b=Advanced Swamp Seed Extraction
+[entity-name]
+algae-farm=Algae Farm 1
+algae-farm-2=Algae Farm 2
+algae-farm-3=Algae Farm 3
+
+crop-farm=Basic Farm
+temperate-farm=Temperate-environment farm
+desert-farm=Desert-environment farm
+swamp-farm=Swamp-environment farm
```

The entity you name `Foo-environment farm` while the upgrade kits you call `Foo environment kit`. I would go with `Foo environment Bar` between those two...

```suggestion
temperate-farm=Temperate environment farm
desert-farm=Desert environment farm
swamp-farm=Swamp environment farm
```

---

This was intentional: (temperate-environment) farm, vs temperate (environment kit). But if you prefer, I'm happy for you to go with your suggestion.

---

ok, didn't think about that, it's fine as is.